app/css/base.scss

```diff
@@ -0,0 +1,158 @@
+$lightgrey: #EDEBEB;
+$darkgrey: #666565;
+$grey: #9C9A9A;
+$base-color: #666666;
+$active: #47a;
+$focus: $base-color;
+$error: #f44;
+$success: #7f7;
+
+$padding: 0.5em;
+
+@mixin headline($size, $color: $base-color) {
+  color: $color;
+  font-size: $size;
+}
+@mixin magic-pad {
+  box-sizing: border-box;
+  padding: $padding;
+}
+@mixin bubble {
+  @include magic-pad;
+  border-radius: 0.25em;
+  font-family: sans-serif;
+}
+@mixin push--auto {
+   margin: {
+     left: auto;
+     right: auto;
+   }
+ }
+ @mixin cf {
+
+	&::after {
+		content: '';
+		display: table;
+		clear: both;
+	}
+}
+@mixin code {
+  background-color: $darkgrey;
+  color: $lightgrey;
+}
+
+$start: 3em;
+@each $h in h1, h2, h3, h4, h5, h6{
+    #{$h} {
+      font-size: $start;
+      }
+      $start: $start - 0.3;
+  }
+* {
+  position: static;
+  width: 100%;
+  margin: 0px;
+  padding: 0px;
+  font-size: 1em;
+  font-weight: normal;
+  line-height: 1.2em;
+  color: $base-color;
+  &:focus {
+    outline: 0;
+  }
+}
+
```

Comment:
**TODO FOR LOST POINTS**

Great job for a robust base.scss, but you also need one more default style for ``label`` per the lab instructions.